packages/react-components/components/MessagingCard.tsx

```diff
@@ -0,0 +1,14 @@
+import Card, { Props } from '@celo/react-components/components/Card'
+import React from 'react'
+import { StyleSheet } from 'react-native'
+
+// Card used by all messaging cards
+export default function MessagingCard({ style, ...props }: Props) {
+  return <Card style={[styles.container, style]} rounded={true} {...props} />
+}
+
+const styles = StyleSheet.create({
+  container: {
+    height: 144,
```

Comment:
I think its best to avoid an explicit height unless there is no other solution. if we must have one consider a min-height. better to have height be determined by content + padding. 

explicit height  can lead to overflow due to differences in language content length, or user chosen font size.

Reply:
I agree with you 💯 

Here's a special case because those cards are displayed in a carousel in the home feed and we need all of them to have the same height so it doesn't look weird when swiping through them.
I did that mostly as a shortcut for now. 

I'll see if I can make the carousel cards grow to reach the height of the longest card.